includes/polling/os/stormshield.inc.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+use LibreNMS\RRD\RrdDefinition;
+
+$hardware = trim(snmp_get($device, '.1.3.6.1.4.1.11256.1.0.1.0', '-Osqv'), '"');
```

Comment:
Can you not use the MIB here to use named OIDs?

Reply:
You should also update all of these to a single `snmp_get_multi_oid` to avoid multiple lookups